package.json

```diff
@@ -43,23 +43,23 @@
     "apollo-client": "^2.4.5",
     "apollo-link-context": "^1.0.9",
     "apollo-server": "^2.0.5",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
```

Comment:
We should consider retaining the [`babel-core@bridge`](https://www.npmjs.com/package/babel-core/v/7.0.0-bridge.0) for things that may depend on it.